### PR TITLE
Landing page of eCommerce

### PR DESCRIPTION
**Pull request by Adarsh-Patel-Dev**
image

image

### CODE REVIEW

File: landing-page/landing.html [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+            <div class="nav--header">
+                <h2 class="nav--title">Plan<span style="color: white;">e</span>t</h2>
+            </div>
+            <div class="nav--search">
+                <input type="text" placeholder="Search here" class="nav--search--input">
+                <button class="nav--search--btn">
+                    <i class='fas fa-search' style='color: white'></i>
+                </button>
+
+
```
Comment (sri-rishi) [inline landing-page/landing.html:34]: remove extra spaces from file

File: landing-page/landing.css [hunk 2]
```diff
@@ -0,0 +1,566 @@
...
+}
+
+.hero-text {
+    width: 780px;
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 123px;
+    line-height: 162px;
+    color: #000000;
```
Comment (sri-rishi) [inline landing-page/landing.css]: be consistent with color value, use variable color naming

File: landing-page/landing.css [hunk 3]
```diff
@@ -0,0 +1,566 @@
...
+.card--details{
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    margin-top: 1rem;
+    padding: 5px 0;
+}
+
+.img-fluid{
+    border-radius: 5px 5px 0px 0px;
```
Comment (sri-rishi) [inline landing-page/landing.css]: Just using 0 instead of 0px will work

**Review by sri-rishi — CHANGES_REQUESTED** [review]

**Review by sri-rishi — COMMENTED** [review]
image

Just a suggestion here color has to much contrast

**Review by JuvitaSaini09 — APPROVED** [review]
your code looks clean and UI is also nice

**Comment by prankurpandeyy** [comment]
looks good bro , you could have made seperate page for product listing ?

**Review by prankurpandeyy — APPROVED** [review]

**Comment by Adarsh-Patel-Dev** [comment]
> looks good bro , you could have made seperate page for product listing ?

Yeah there will be a seperate product listing page

File: landing-page/landing.css (hunk 3 above)
Comment (Adarsh-Patel-Dev) [reply landing-page/landing.css]: ok I'll fix it

File: landing-page/landing.css (hunk 2 above)
Comment (Adarsh-Patel-Dev) [reply landing-page/landing.css]: ok forgot to use var here

**Pull request merged**